src/config.cpp

```diff
@@ -421,6 +421,7 @@ namespace config {
     },  // supported resolutions
 
     { 10, 30, 60, 90, 120 },  // supported fps
+    0, // max bitrate
```

Comment:
```suggestion
    0,  // max bitrate
```

2 spaces (linting rules)